src/helpers/constants.ts

```diff
@@ -0,0 +1 @@
+export const AMAZON_US_MARKETPLACE_ID = 'ATVPDKIKX0DER'
```

Comment:
I feel like this should not be a constant in this lib.

Reply:
Its the default Amazon US marketplace, It's safe to be here.